modules/serverless-create-broker-kn.adoc

```diff
@@ -16,12 +16,21 @@ Brokers can be used in combination with triggers to deliver events from an event
 
 .Procedure
 
-* Create the `default` broker:
+* Create a broker:
 +
 [source,terminal]
 ----
-$ kn broker create default
+$ kn broker create <broker_name>
 ----
++
+The previous command creates a broker that uses the default broker class that is configured for the cluster or namespace. If you want to use a different broker class, you can do this by specifying the `--class` flag:
++
+[source,terminal]
+----
+$ kn broker create <broker_name> --class <broker_class>
```

Comment:
Not exactly sure how the `kn client` does this, but usually the `class` on it is own is not enough. See here for `class` and additional config-map:

https://github.com/knative-sandbox/eventing-kafka-broker/blob/main/test/config/config-br-defaults.yaml#L25-L29

/cc @dsimansk

Reply:
Yes, that's why the docs say "See the documentation about "Configuring the default broker class" in the "Additional resources" section for more information on which broker class options can be configured if you have administrator permissions." and then there is a link in the additional resources to the docs for the config map?

Reply:
This example is correct. However, Matthias is also right that it might not cover all the use cases. Maybe for now I'd just keep that `kn broker create <broker_name>` cmd can create a default broker. And skip `--class` until later releases. 

We have a flag for additional broker spec modification, e.g. when ConfigMap is needed to be referenced. However, it's pretty new and it will be available in Serverless 1.27.
```
kn broker create mybroker --namespace myproject --class Kafka --broker-config cm:spec-cm
```